core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -55,10 +63,16 @@
 
     @JsonIgnore
     private List<Map<String, Object>> adapterConfigurations;
+
+    @JsonIgnore
+    private List<TrustAnchor> trustAnchors;
+
     @JsonIgnore
     private ResourceLimits resourceLimits;
     @JsonIgnore
     private TrustAnchor trustAnchor;
```

Comment:
what is this good for?

Reply:
I might have mistakenly left it there after a rebase to resolve conflicts. will remove